modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -270,14 +269,7 @@ const RichText = ({
 					{Liferay.FeatureFlags['LPD-11235'] ? (
 						<CKEditor5ClassicEditor
 							className="w-100"
-							config={{
-								filebrowserImageBrowseUrl:
-									editorConfig.filebrowserImageBrowseUrl,
-								filebrowserVideoBrowseUrl:
-									editorConfig.filebrowserVideoBrowseUrl,
-								itemSelectorEventName: `${name}selectItem`,
-								preset: EClassicEditorConfigPreset.ADVANCED,
-							}}
+							config={editorConfig}
```

Comment:
We don't need anything special for CKE5, that is why there is no new `CKEditor5EditorConfigContributor`.

We are still getting here:
- `preset` and `itemSelectorEventName` from base [CKEditor5ClassicEditorConfigContributor](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditor5ClassicEditorConfigContributor.java)
- `file*` item selector URLs from [item selector contributors](https://github.com/liferay/liferay-portal/tree/master/modules/apps/item-selector/item-selector-editor-configuration/src/main/java/com/liferay/item/selector/editor/configuration/internal), that affect all editors